financeager/__init__.py

```diff
@@ -3,7 +3,11 @@
 from logging import getLogger, StreamHandler, DEBUG, INFO, Formatter, handlers
 
 # versioning information
-__version__ = "0.21"
+from pkg_resources import get_distribution, DistributionNotFound
+try:
+    __version__ = get_distribution(__name__).version
+except DistributionNotFound:
+    __version__ = "0.22"
```

Comment:
Why did you add this as fallback?

Reply:
Do you need 0.22 as fallback version ?
I shortcut output from python setup.py --version but you are right, I'm worng it should be 0.21
Should I provide a new pull request to correct default number version ?
Alos if I provide a new pull rquest, should I squash these commits ?

Reply:
I'd like to avoid having to manually update version information in the source code, so I'd omit handling DistributionNotFound (the version info does get lost when the code is torn out of the context of distribution using pip/setup.py or git, e.g. if anyone was to copy it around to different machines; but hey it's 2019 and package distribution is supposed to be done using pip, and package development is supposed to be done using git ;))

You could amend your commit to add the correction (do the change, `git add` it, and run `git commit --amend`. Then you have to force-push to overwrite the commit on the remote). This means you don't have to open another PR, and the current one still holds only one commit.

Reply:
Ok it's clear, thanks